services/s3/src/it/java/software/amazon/awssdk/services/s3/crossregion/S3CrossRegionCrtIntegrationTest.java

```diff
@@ -46,7 +46,7 @@ static void clearClass() {
     @BeforeEach
     public void initialize() {
         crossRegionS3Client = S3AsyncClient.crtBuilder()
-                                           .region(CROSS_REGION)
+                                           .region(Region.AWS_GLOBAL)
```

Comment:
Intentional change?

Reply:
yeap , making sure we test this in Integ too since the cross region feature depends mainly on the error headers send by S3 based on the client region configured.
We have different client configured in different regions in https://github.com/aws/aws-sdk-java-v2/tree/b415e16e755d7c95c7bfae4725de4d623fb43c5b/services/s3/src/it/java/software/amazon/awssdk/services/s3/crossregion